docs/examples/irradiance-transposition/plot_rtranpose_year.py

```diff
@@ -0,0 +1,134 @@
+"""
+Reverse transposition using one year of hourly data
+===================================================
+
+With a brief look at accuracy and speed.
```

Comment:
```suggestion
When irradiance is measured on a tilted plane, it is useful to be able to estimate
the GHI that produces the POA irradiance. The estimation requires inverting a GHI-to-POA
irradiance model, which involves two parts: a decomposition of GHI into direct and diffuse components,
and a transposition model that calculates the direct and diffuse irradiance on the tilted plane.
Recovering GHI from POA irradiance is termed "reverse transposition."
```

Reply:
Thanks for the introductory text.  I added it.